cmd/zstreamdump/zstreamdump.c

```diff
@@ -509,6 +514,7 @@ main(int argc, char *argv[])
 			}
 
 			payload_size = DRR_WRITE_PAYLOAD_SIZE(drrw);
+			drr_byte_count[drr->drr_type] += payload_size;
```

Comment:
This is not a huge deal, but it might be simpler to set payload_size=0 first thing inside the while loop, and then have each record type set payload_size, and then after the switch statement increment drr_byte_count[].  That way you only have one place that's incrementing the drr_byte_count.